### ISSUE TITLE
What does shrink mean in the test.py?

### ISSUE THREAD

**Issue by Edwardmark** [Goingqs/PyramidBox#13]
What does shrink mean in the test.py? 
  
https://github.com/Goingqs/PyramidBox/blob/master/test.py#L231

why divide 0x7fffffff by 200 and the image shape?

**Comment by Edwardmark** [comment]
@Goingqs Please comment, thank you very much.

**Comment by hdjsjyl** [comment]
@Edwardmark "divide 0x7fffffff by 200" is the limitation of image size in caffe

**Comment by Goingqs** [comment]
@hdjsjyl That's it.

**Comment by Edwardmark** [comment]
@hdjsjyl @Goingqs Thanks, guys.

**Comment by hdjsjyl** [comment]
@Edwardmark  Are you successful to train the model which can have the same performance of model provided by @Goingqs ? If you are successful, can you share your environment and parameters setting including pytorch version, cuda version, cudnn version, batch size, weight decay and anything you changed? Thanks
